Correct spelling mistake on AAD Group expiration

BrianTJackett opened the pull request:
Corrected "t hegroup" to "the group" in Azure AD documentation on Office 365 Groups expiration policy.

azurecla commented:
Hi __@alexandair__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

Jim-Parker commented:
@markgalioto, Mark, can you take a look at Aleksandar's contributions to your article and let me know if they are okay to merge? Please let him know as well. Thanks,

Jim-Parker commented:
@alexandair Thanks for the contribution Aleksandar. I'll merge these updated today.

BrianTJackett committed:
Correct spelling mistake on AAD Group expiration

Corrected "t hegroup" to "the group" in Azure AD documentation on Office 365 Groups expiration policy.
